path_finding.asciidoc

```diff
@@ -223,6 +222,49 @@ Once we attempt to send an HTLC we can learn more about channel balances: if we
 You may be thinking "What's the point of learning from a successful HTLC?" After all, if it succeeded we're "done". But consider that we may be sending one part of a multi-part payment. We also may be sending other single-part payments within a short time. Anything we learn about liquidity is useful for the next attempt!
 ====
 
+==== Liquidity uncertainty and probability
+
+To quantify the uncertainty of a channel's liquidity, we can apply probability theory. A basic model of the probability of payment delivery will lead to some rather obvious, but important, conclusions:
+
+* Smaller payments have a better chance of successful delivery across a path.
```

Comment:
...across a fixed path....

we don't loose anything by adding the adjective but it is actually more acurate. because depending on the path there might be paths where larger payments have a higher success chance the other paths with lower amounts.